navigator_engine/model.py

```diff
@@ -20,7 +20,6 @@ def to_networkx(self):
 
 class Conditional(db.Model):
     id = db.Column(db.Integer, primary_key=True)
-    ref = db.Column(db.String)
```

Comment:
@jsoppela Please confirm that this column is not actually needed in the model?

Reply:
We are not using the ref column in the database at the moment but the data frame in graph loader uses it as index column. I think storing it is still a good idea.